### DISCUSSION TITLE
LibreOffice freezes randomly when window is moved with Rectangle

### DISCUSSION THREAD

**Discussion by kolydart** [rxhanson/Rectangle#1048] [General]
There is a bug in LibreOffice & Rectangle which makes the former to freeze when a window is moved. 
I have not been able to reproduce the behavior, thus not been able to submit an issue in any of the two applications.
I start a discussion here in case more people face the same problem in order to coordinate tracing the cause.
For the moment I can tell it happens frequently, but randomly when I maximize a LibreOffice document. It does not happen immediately after creating a new document.

**Comment by Aaron-Rumpler** [comment]
Sounds like #933.

**Reply by kolydart** [reply]
Thanks for the indication, @Aaron-Rumpler . Yes, it looks like it is a duplicate of #933. I searched for active issues but did not check for closed ones.
